Spanner/src/Operation.php

```diff
@@ -561,7 +562,8 @@ public function createSession($databaseName, array $options = [])
         $res = $this->connection->createSession([
             'database' => $databaseName,
             'session' => [
-                'labels' => $this->pluck('labels', $options, false) ?: []
+                'labels' => $this->pluck('labels', $options, false) ?: [],
+                'creator_role' => $this->pluck('creator_role', $options, false) ?: null
```

Comment:
Cloud Spanner databases come with a few predefined system roles, such as `public` and a few others like `spanner_<something>`. I think we need to confirm whether a default `creator_role` is supposed to go as `null` or `public`

Reply:
As far as I experimented, we can choose to not pass the creator_role key, or pass the null value for the key.